src/demo/rewireEnterpriseDataApiService.test.js

```diff
@@ -102,7 +102,7 @@ describe('rewireEnterpriseDataApiService', () => {
         },
         course_completions: expect.any(Number),
         enrolled_learners: expect.any(Number),
-        last_update_date: expect.any(Date),
+        last_updated_date: expect.any(String),
```

Comment:
my bad, missed this file :grimacing: 

However, strange how did this test pass :confused: ?

Reply:
@mushtaqak Test didn't pass, but was reported incorrectly. See the link I posted above. Will look into why these errors aren't failing the build 😠